src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -125,4 +132,28 @@ private function findRouteNameContaining(string $name, RouteCollection $routes):
 
         return $foundRoutesNames;
     }
+
+    private function sortRoutes(RouteCollection $routes, string $propertyName): RouteCollection
+    {
+        $validOptions = ['', 'priority', 'name', 'path'];
+        $sortedRoutes = $routes->all();
+        if ('name' === $propertyName) {
+            ksort($sortedRoutes);
+        } elseif ('path' === $propertyName) {
+            uasort(
+                $sortedRoutes,
+                static function (Route $a, Route $b): int {
+                    return $a->getPath() <=> $b->getPath();
+                }
+            );
+        } elseif (!\in_array($propertyName, $validOptions)) {
```

Comment:
`} else {`

Reply:
I have to take care of `sortOption = 'priority'`, so I removed the `$validOptions` array and changed the last `elseif` to        
```
 } elseif ('priority' !== $propertyName) {
            throw new InvalidArgumentException...
}
```